Birdui/Birdui/PostListView.swift

```diff
@@ -9,10 +9,56 @@
 import SwiftUI
 
 struct PostListView: View {
+    @State var modalIsPresented = false
+
+    struct HeaderTitleTextStyle: ViewModifier {
+        func body(content: Content) -> some View {
+            return content
+                .foregroundColor(Color.black)
+                .font(.system(size: 17))
+        }
+    }
+
+    struct ButtonTextStyle: ViewModifier {
+        func body(content: Content) -> some View {
+            return content
+                .font(.system(size: 15))
+        }
+    }
+
     var body: some View {
-        // TODO: This should look exactly like the Birdie table view,
-        // but with only one button.
-        Text("Layout header, new-post button, List of posts")
+        NavigationView {
+        VStack(alignment: .leading) {
+            let leftImageSize = 50
+            HStack {
+                Image("mascot_swift-badge")
+                    .resizable()
+                    .frame(width: 50, height: 50)
+                Spacer()
+                Text("Home")
+                    .modifier(HeaderTitleTextStyle())
+                    .padding(.leading, -CGFloat(leftImageSize))
```

Comment:
You used `leftImageSize` only in one place, but I think you meant to use it in `Image.frame` as well, didn't you?

Reply:
the leftImageSize constant is to substract the width of the image in order to center the "Home" text in the HStack line 40